cpu/cc2538/include/cpu_conf.h

```diff
@@ -43,6 +43,19 @@ extern "C" {
 #define CPU_HAS_BITBAND                 (1)
 /** @} */
 
+/**
+ * @name    OpenWSN timing constants
+ *
+ * @{
+ */
+/* Taken from openwsn-fw */
+#define PORT_maxTxDataPrepare   (460/PORT_US_PER_TICK)
+#define PORT_maxRxAckPrepare    (300/PORT_US_PER_TICK)
+#define PORT_maxRxDataPrepare   (300/PORT_US_PER_TICK)
+#define PORT_maxTxAckPrepare    (460/PORT_US_PER_TICK)
+#define PORT_delayTx            (400/PORT_US_PER_TICK)
```

Comment:
Does it make sense to rework these to RIOT-specific defines and keep the openwsn-specific defines in the package?

So here an:
```
#define RADIO_MAX_TX_DATA_PREPARE   (460/PORT_US_PER_TICK)
#define RADIO_MAX_RX_ACK_PREPARE    (300/PORT_US_PER_TICK)
#define RADIO_MAX_RX_DATA_PREPARE   (300/PORT_US_PER_TICK)
#define RADIO_MAX_TX_ACK_PREPARE    (460/PORT_US_PER_TICK)
#define RADIO_DELAY_TX                     (400/PORT_US_PER_TICK)
```

And in the OpenWSN define the OpenWSN-specific defines to these values. This way we can easily reuse these values for other MAC protocols.

Reply:
Not really, maybe the ls two `RADIO_DELAY_TX` and `RADIO_DELAY_RX`, the other names are specific to OpenWSN state machine, its a mix of hardware and code introduced delays https://openwsn.atlassian.net/wiki/spaces/OW/pages/688255/Timing+Constants.

Reply:
Are this CPU configurations or board configurations?

Reply:
They should be CPU configuration in the nrf52840 case right? I think the same holds for the cc2538, or any CPU with integrated radio.

Reply:
> Are this CPU configurations or board configurations?

CPU since its an integrated radio, for the `iotlab-m3` and `samr21-xpro` they are BOARD configurations.

Reply:
at some point we would like to have a way to ask the hardware about its timings. Maybe that should be part of the Radio HAL, but we are quite far from there (: